### PR TITLE
[test-only] add test to filter or

### PR DESCRIPTION
**Pull request by ScharfViktor**
I added test for filter `or` https://github.com/owncloud/ocis/pull/5694

### CODE REVIEW

**Comment by ownclouders** [comment]
E2E tests failed: https://drone.owncloud.com/owncloud/ocis/20069/64/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/ocis/20069/tracing/file-alice-2023-3-8-02-35-00.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/ocis/20069/tracing/user-login-can-be-managed-in-the-admin-settings-admin-2023-3-8-02-31-10.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/ocis/20069/tracing/user-login-can-be-managed-in-the-admin-settings-admin-2023-3-8-02-31-15.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/ocis/20069/tracing/user-login-can-be-managed-in-the-admin-settings-alice-2023-3-8-02-31-14.zip 


</p></details>

File: tests/acceptance/features/apiGraph/getUser.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@api @skipOnOcV10
+@api @skipOnOcV10 @skipOnStable2.0
```
Comment (SagarGi) [inline tests/acceptance/features/apiGraph/getUser.feature:1]: I think the other scenarios for this feature runs on `stable-2.0`. My be you should put it on your newly added scenrios which donot run on `stable-2.0`

File: tests/acceptance/features/apiGraph/getUser.feature [hunk 2]
```diff
@@ -135,6 +135,26 @@ Feature: get users
       | Alice Hansen | %uuid_v4% | alice@example.org | Alice                    | true           |
       | Carol King   | %uuid_v4% | carol@example.org | Carol                    | true           |
 
+  
+  Scenario: admin user gets all users of certain groups
```
Comment (SagarGi) [inline tests/acceptance/features/apiGraph/getUser.feature:139]:
```suggestion
  @skipOnStable2.0
  Scenario: admin user gets all users of certain groups
```

File: tests/acceptance/features/apiGraph/getUser.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/acceptance/features/apiGraph/getUser.feature:1]: yes, you are right

**Review by individual-it — APPROVED** [review]

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Pull request merged**